examples/jenkins/README.md

```diff
@@ -19,6 +19,10 @@ Steps
 
     This will allow the service account Jenkins uses to run builds and tag images.
 
+    However, if you are working with the OpenShift Online environment, you will need to update the `openshift-online` user instead:
```

Comment:
@liggitt 's concerns still pending, I pushed an update based on @csrwng 's comment (though I tweaked it slightly so that the verb used was more consistent with the prior bullet).